Build-Core.cmd

```diff
@@ -2,7 +2,7 @@
 setlocal
 
 SET CMDHOME=%~dp0.
-SET BUILD_FLAGS=/m:1 /v:m /fl /flp:logfile=build.log;verbosity=detailed
```

Comment:
You can bring this back as @jdom mentions, just change the verbosity from `detailed` to `normal` so it's not as noisy.